wordpress-mobile/WordPress-iOS
# Issue/clean up media removal and cleanup code

Remove the cleanup media code, because besides not working properly is no longer needed because any media that is not upload to the server is still usable trough the media library. 
I also improve the removal of files when a media object is removed in order to not show error messages if the files where already remove in a previous step.

To test: 
 - Create a post
 - Add media to it (images or video), cancel the uploads on the middle of the process
- Check if no  logging messages are show referring to unable to removal files.
- If need check the Media folder inside the documents folder of the app to see if the files associated are removed.

Needs review: @astralbodies
